CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
   `Html::h5()`, `Html::h6()` (vjik)
 - New #100: Add classes for tags `Picture`, `Audio`, `Video`, `Source` and `Track` (Gerych1984, vjik)
 
+- New #89: Add method `nofollow()` to the `A` tag (soodssr)
```

Comment:
This is artefact of merging. Need remove this line.

Can you add line about changes?

Reply:
sure